sml_builder/utils.py

```diff
@@ -27,6 +28,25 @@ def get_feature_config(feature_name: str):
         raise KeyError(f"Feature '{feature_name}' not found in features.") from e
 
 
+# For use with code optioning. This function reads the state of the specified feature from the feature.json file and updates it.
+def update_feature_config(feature_name: str, feature_value: bool):
```

Comment:
Where is this function used?

Reply:
This was used so that a function could show/hide a feature when needed. It was something we used to use for the search tool and now is no longer needed. I will remove it, I kept it in as I thought it may be useful in future if a function needed to disable/hide or enable/show features.